src/applications/gi/components/vet-tec/EybVetTecApprovedPrograms.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ContactInformation from '../profile/ContactInformation';
+import { formatCurrency, isPresent } from '../../utils/helpers';
+import { connect } from 'react-redux';
+
+class EybVetTecApprovedPrograms extends React.Component {
+  render() {
+    const programs = this.props.institution.programs;
+    const selectedProgram =
+      this.props.selectedProgram !== ''
+        ? this.props.selectedProgram
+        : this.props.preSelectedProgram;
+
+    if (programs && programs.length) {
+      const programRows = programs.map((program, index) => {
+        const programLength =
+          isPresent(program.lengthInHours) && program.lengthInHours !== '0'
+            ? `${program.lengthInHours} hours`
+            : 'TBD';
+        const tuition = isPresent(program.tuitionAmount)
+          ? formatCurrency(program.tuitionAmount)
+          : 'TBD';
+        const checked =
+          selectedProgram &&
+          program.description.toLowerCase() === selectedProgram.toLowerCase();
+        return (
+          <tr key={index}>
+            <td className="vads-u-padding-left--0 vads-l-grid-container">
+              <div className="program-description vads-l-row">
+                <div className="vads-l-col--2 checked-program">
+                  <i
+                    className={
+                      checked
+                        ? 'fas fa-check vads-u-padding-right--0p5 vads-u-color--green'
+                        : 'vads-u-padding-right--2p5'
+                    }
+                  />
+                </div>
+                <div className="vads-l-col--10 ">
+                  {program.description}{' '}
+                  {checked ? (
+                    <span className="sr-only">selectedProgram</span>
```

Comment:
Should the be "Selected Program" with a space in between the words?

Reply:
Done.